### ISSUE TITLE
More support for editing polygons/multigeometries?

### ISSUE THREAD

**Issue by nilsnolde** [opengisch/QField#3682]
Is there plans to add the ability to 
- add inner rings for *polygons?
- add sub-features to multigeometries? Seems like currently it'll add a whole new multi-feature and one can't append to existing ones.

Hope I didn't miss anything obvious and those things are really missing. If not, would be happy with a PR/funding to support that? I'm just researching a project proposal and in case that'd be an option, we could plan for it. This would only really become a topic if the proposal would be accepted. It'd just be good to know if that's an option while preparing it.

**Comment by nirvn** [comment]
@nilsnolde , adding inner rings to single polygons is already possible using the geometry editor; I'll rephrase this to refer to a generic "multi geometry support" enhancement request.

That said, it's unlikely to happen without people / groups stepping up to support an implementation.
